### PR TITLE
Update HTML files with Code Smells in Sonar.

### PR DESCRIPTION
**Pull request by ajtortolero**
Some files are reported in sonar with code smells, mostly the report is that they have characters longer than 120

This update is to improve these code-smells reported in sonar.

### CODE REVIEW

File: dojo/templates/dojo/components.html [hunk 1]
```diff
@@ -11,7 +11,12 @@
                 <h3 class="has-filters">
                     Components
                     <div class="dropdown pull-right">
-                        <button id="show-filters" data-toggle="collapse" data-target="#the-filters" class="btn btn-primary toggle-filters"> <i class="fa-solid fa-filter"></i> <i class="caret"></i> </button>
+                        <button id="show-filters"
```
[the remark this answers is not in the document]
Comment (ajtortolero) [reply dojo/templates/dojo/components.html:14]: This update is to improve these code-smells reported in sonar.

File: dojo/templates/dojo/apply_finding_template_form_fields.html [hunk 2]
```diff
@@ -1,111 +1,111 @@
...
+    {{ field }}
+{% endfor %}
+
+{% for field in form.visible_fields %}
+    {% with template_value=template|get_attribute:field.name|default_if_none:'' %}
+    <div class="form-group{% if field.errors %} has-error{% endif %}">
+        <label class="col-sm-2 control-label">{{ field.label }} Option</label>
+        <div class="col-sm-10 {{ classes.value }}">
+            {% if field.name == 'tags' %}
+                <select class="form-control" data-field-value="{{ finding_tags|as_json }}" data-template-value="{{ field.value|as_json }}" onchange="(function(t){ var field = document.forms['apply_template']['{{ field.name }}']; var fieldValue = ''; switch(t.value.toLowerCase()){ case 'keep': fieldValue=JSON.parse(t.getAttribute('data-field-value')); break; case 'replace': fieldValue= JSON.parse(t.getAttribute('data-template-value')); break; case 'combine': fieldValue= JSON.parse(t.getAttribute('data-field-value')).concat(JSON.parse(t.getAttribute('data-template-value'))); break;  } $(field).val(fieldValue); })(this)">
```
[the remark this answers is not in the document]
Comment (ajtortolero) [reply dojo/templates/dojo/apply_finding_template_form_fields.html:32]: This update is to improve these code-smells reported in sonar.

**Comment by Maffooch** [comment]
Looks like the accessibility violations were in place before the linter was enforced. Please update to be in compliance with AccessLint

**Review by Maffooch — DISMISSED** [review]

**Comment by Maffooch** [comment]
Closing due to stale

**Pull request closed without merging**